packages/core-mobile/app/new/routes/(signedIn)/(modals)/accountSettings/manageAccounts.tsx

```diff
@@ -105,104 +121,214 @@ const ManageAccountsScreen = (): React.JSX.Element => {
     [dispatch]
   )
 
-  const walletsDisplayData: (WalletDisplayData | null)[] = useMemo(() => {
-    const walletArray = Object.values(allWallets)
-    return walletArray
-      .map(wallet => {
-        const accountsForWallet = accountSearchResults.filter(
-          account => account.walletId === wallet.id
-        )
-
-        if (accountsForWallet.length === 0 && searchText) {
-          return null
-        }
+  const privateKeyWallets = useMemo(() => {
+    return Object.values(allWallets).filter(
+      wallet => wallet.type === WalletType.PRIVATE_KEY
+    )
+  }, [allWallets])
+
+  const otherWallets = useMemo(() => {
```

Comment:
```suggestion
  const primaryWallets = useMemo(() => {
```
The name `otherWallets` makes these wallets sound less important or secondary. 😅